R/class-Comparator.R

```diff
@@ -0,0 +1,78 @@
+
+check_comparator <- function(object) {
```

Comment:
the comparator class could probably go in veupathutils, but im happy to leave it here until we def need it somewhere else.

Reply:
nothing specific, just noticing this isnt really a mbio specific class